iot/api-client/mqtt_example/cloudiot_mqtt_example.py

```diff
@@ -169,37 +169,200 @@ def get_client(
 # [END iot_mqtt_config]
 
 
+def detach_device(client, device_id, mqtt_bridge_hostname, mqtt_bridge_port):
+    """Detach the device from the gateway."""
+    # [START detach_device]
+    detach_topic = '/devices/{}/detach'.format(device_id)
+    print('Detaching: {}'.format(detach_topic))
+    client.loop()
+    client.connect(mqtt_bridge_hostname, mqtt_bridge_port)
+    client.publish(detach_topic, '{}', qos=1)
+    time.sleep(5)  # wait for the server to respond / will trigger callback
+    # [END detach_device]
+
+
+def attach_device(client, device_id, mqtt_bridge_hostname, mqtt_bridge_port):
```

Comment:
`mqtt_bridge_*` variables seem unused here. Seems like `attach` and `detach` are not symmetrical, though I'm implementing it as symmetrical in other languages.

Reply:
I wanted for these to be usable as stand-alone functions, if we don't want the connect call, lmk, the client can disconnect on error before the attach call.